public/src/components/Spinner/Spinner.js

```diff
@@ -0,0 +1,29 @@
+import React, {useState} from 'react'
+import {useSelector} from "react-redux";
+import Loader from "react-loader-spinner";
+
+
+function Spinner(props) {
+
+    console.log ('props', props)
+
+    return (
+        <div>
+            
+            {props.loaded === false ? (
```

Comment:
It might make more sence to call this prop name something other than loaded. Maybe `display` or `showSpinner` loading is specific to what uses it

Reply:
is loaded is for the page. page is loaded or not loaded.